examples/evm-to-evm-fungible-transfer/src/transfer.ts

```diff
@@ -1,5 +1,14 @@
 import { EVMAssetTransfer, Environment } from "@buildwithsygma/sygma-sdk-core";
 import { Wallet, providers } from "ethers";
+import * as dotenv from "dotenv";
```

Comment:
you can do `import dotenv from 'dotenv'`

Reply:
fixed